_data/projects/RescueApp.yml

```
@@ -0,0 +1,13 @@
+---
+name: RescueApp
+desc: Web application designed to help emergency services in registering and documenting tasks to rescue lost people.
+site: https://github.com/eduayme/RescueApp
+tags:
+- web
+- php
+- laravel
+- postgresql
+- community
+upforgrabs:
+  name: Help wanted
```

This needs to match exactly with the label used on the project to light up the stats for the project:

```suggestion
  name: help wanted
```